saifa/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  backend:
+    image: patranakamo/saifa-be:v1.3.3
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "9988:80"
```

Comment:
Do you need this port bound to the host? This would be necessary if external applications are trying to send requests directly to this backend container.

Or is it just your frontend container that is connecting to the backend? If so, then there is no need to bind the port to the host, since your frontend container can resolve the ip of the backend container using Docker's inbuilt dns and communicate on container port 80 directly.